examples/remote_query/template/README.md

```diff
@@ -0,0 +1,9 @@
+# Mephisto Remote Query Template
+
+This task is an _extremely rough_ bare-bones example of being able to set up a task where the frontend directly connects to the backend using the `mephisto-remote-query` package. It should serve as a decent example of how to get a `RemoteQueryBlueprint` task up off the ground.
```

Comment:
need to update now that there's no `mephisto-remote-query` package

Reply:
Ah right! Spent a lot of time testing MNIST and less on the template...